babel/src/test/java/org/apache/calcite/test/BabelParserTest.java

```diff
@@ -328,6 +328,54 @@ private void checkParseInfixCast(String sqlType) {
     sql(sql).ok(expected);
   }
 
+  @Test public void testTableAttributeJournalDefault() {
```

Comment:
testTableAttributeJournalDefault can be change to testTableAttributeJournal_defaultModifier_defaultType